pavelib/assets.py

```diff
@@ -64,10 +64,6 @@
     'requirejs/require.js',
     'underscore.string/dist/underscore.string.js',
     'underscore/underscore.js',
-    '@edx/studio-frontend/dist/assets.min.js',
```

Comment:
I discovered that we need to also fix this file to get the edx namespace to be picked up. Would you be able to include this change too:

    # Common lookup paths that are added to the lookup paths for all sass compilations
    COMMON_LOOKUP_PATHS = [
        path("common/static"),
        path("common/static/sass"),
        path('node_modules/@edx'),
       path('node_modules'),
    ]

Reply:
Roger, I'll add it right now